src/lib/ekoEmbed/v1.js

```diff
@@ -62,12 +64,16 @@ class EkoEmbedV1 {
         this.eventEmitter.once(eventName, callback);
     }
 
+    dispose() {
+        window.removeEventListener('message', this.onEkoEventFired);
```

Comment:
Would recommend refactoring this into a `removeIframeListeners()` private method.
Then the entire implementation of dispose would be:

```js
dispose() {
    this.removeIframeListeners();
}
```

The fact that currently the adding of events is done in a named function, while removing is done directly in `dispose()` may cause bugs as code evolves (it would be easier to add a listener but forget to remove it).